### ISSUE TITLE
Build specs seem to not reflect actual build spec used

### ISSUE THREAD

**Issue by mbhall88** [singularityhub/singularityhub.github.io#178]
## Link to Container Collection Log, Build, or Collection (in that order)
https://singularity-hub.org/containers/9401  
https://singularity-hub.org/containers/9402

### Behavior when Building Locally

Builds ok.

### Error on Singularity Hub

The containers build fine on Singularity Hub and runs as expected when pulled and executed locally.

### What do you think is going on?

The build spec for these two containers seems to show the recipe from the previous commit. However, the container that is actually built reflects the real build spec on Github. I guess this isn't a major issue, just thought I would flag it.  
The different in the two is the `VERSION` variable. It shows `3.1.5` but in reality it built `3.2.1`, which is what is reflected on the [github build spec](https://github.com/mbhall88/Singularity_recipes/blob/3430d70b617125d712b9f0807b8393bcbf0ea37f/recipes/Singularity.guppycpu#L20)

**Comment by vsoch** [comment]
What builder are you using?

**Comment by vsoch** [comment]
If you use 3.2.1 there is a known bug that it builds only the recipe at the base of the repo. It’s an unannounced testing builder that I need a few hours to debug.

**Comment by vsoch** [comment]
hey @mbhall88 good news! I looked at the log for the container -> https://singularity-hub.org/containers/9402/log and it did look like the correct version was built, and the files in storage are also correct. What happened was a bug with the recipe file sent back - it wasn't matching a pattern so it wasn't being updated (and we saw the old one). I just fixed this on the server, and I also updated those two recipes to reflect what we have in storage. The builder for 3.2.1 is another issue that I still need to work on (but you aren't using it, so not a problem). Let me know if I can help with anything else!
